app/controllers/v1/charts_controller.rb

```diff
@@ -37,6 +37,15 @@ def destroy
     head :no_content
   end
 
+  def clone
+    authorize! :clone, Chart
+
+    chart = Chart.find_by(id: params[:id]).clone
+    chart.update(position: (Chart.where(dashboard_section_id: chart.dashboard_section_id).maximum(:position) || 0) + 1)
```

Comment:
```suggestion
    chart.update(position: (Chart.where(dashboard_section_id: chart.dashboard_section_id).maximum(:position) || 0).next )
```

Reply:
I believe there's one more place when `+ 1` could be replaced with `.next`